WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -197,7 +197,7 @@ class AuthenticationManager: Authentication {
             let viewModel = NoWooErrorViewModel(
                 siteURL: siteURL,
                 showsConnectedStores: matcher.hasConnectedStores,
-                showsInstallButton: matchedSite.isJetpackConnected,
+                showsInstallButton: matchedSite.isJetpackConnected && matchedSite.isJetpackThePluginInstalled,
```

Comment:
Yes, we discovered an issue with Calypso when the Woo installation flow doesn't work properly with JCP sites (p1660123571755889-slack-C02DQP0FP) - so we decided to not support JCP sites.

Reply:
I see now, do you think we can add a comment somewhere about this reason?

Reply:
Since this logic is also used in `StorePickerViewController` https://github.com/woocommerce/woocommerce-ios/pull/7439#discussion_r943035935, maybe a boolean extension on `Site` would be helpful with a comment on why JCP sites aren't supported?

Reply:
I intended to bring this logic to `NoWooErrorViewModel` instead, so I'll make the change as part of the Woo setup enhancement in #7455. How does that sound?